velox/type/TimestampConversion.h

```diff
@@ -107,6 +107,38 @@ Status daysSinceEpochFromWeekDate(
     int32_t dayOfWeek,
     int64_t& out);
 
+/// Computes the signed number of days since the Unix epoch (1970-01-01). To
+/// align with Spark's SimpleDateFormat behavior, this function offers two
+/// modes: lenient and non-lenient. If `lenient` is false, it returns an error
+/// status if the date is invalid. If `lenient` is true, it accepts a wider
```

Comment:
nit:
If `lenient\` is false -> For non-lenient mode
If `lenient` is true -> For lenient mode